### PR TITLE
Category

### PR DESCRIPTION
**Pull request by INR4GE**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "apollo-boost": "^0.4.9",
```
Comment (yaroslavbilokin) [inline package.json:14]: Please remove this, we have to use apollo-client v3 instead of apollo-boost

File: src/configs/index.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
 
 export const config = {
   app: {
     title: 'Horondi Admin Portal',
-    menuCategories: [['Новини', routes.pathToNews, ImportContactsIcon]],
+    menuCategories: [
+      ['Новини', routes.pathToNews, ImportContactsIcon],
+      ['Категорії', routes.pathToCategories, CategoryIcon]
+    ],
     routes,
```
Comment (yaroslavbilokin) [inline src/configs/index.js:22]: please remove this, if you added `export` to routes constant

File: src/configs/index.js [hunk 3]
```diff
@@ -21,8 +29,73 @@ export const config = {
...
+      images: {
+        large: 'as',
+        medium: 'wew',
+        small: 'ee',
+        thumbnail: 'dd'
+      },
+      isMain: true,
+      name: [
+        {
+          lang: 'ru',
```
Comment (yaroslavbilokin) [inline src/configs/index.js:58]: please remove 'ru', we don't have Russian in our project

File: src/configs/index.js [hunk 4]
```diff
@@ -21,8 +29,73 @@ export const config = {
...
+          value: 'rukzak'
+        },
+        {
+          lang: 'eng',
```
Comment (yaroslavbilokin) [inline src/configs/index.js:62]: please use 'en' instead of 'eng'

File: src/pages/categories/categories-page/categories.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    dispatch(getCategories());
+  }, [dispatch]);
+
+  const classes = useStyles();
+
+  if (categoriesLoading) {
+    return <LoadingBar />;
+  }
+
+  if (categoriesError) {
```
Comment (yaroslavbilokin) [inline src/pages/categories/categories-page/categories.js:64]: Maybe moving this to sagas, would be better?

File: src/redux/categories/categories.reducer.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  RESET_NEW_CATEGORY
+} from './categories.types';
+import { config } from '../../configs';
+
+const initialState = {
+  categories: [],
+  categoriesLoading: false,
+  categoriesError: null,
+  newCategory: config.templates.categoryTemplate
+  // newCategory: config.templates.filledCategory,
```
Comment (yaroslavbilokin) [inline src/redux/categories/categories.reducer.js:15]: please remove this

File: src/redux/categories/categories.sagas.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+  }
+}
+
+function* handleCreateCategory({ payload }) {
+  try {
+    yield put(setCategoriesLoading(true));
+    yield call(createCategory, payload);
+    yield put(push('/categories'));
+  } catch (e) {
+    console.log('eeeeeeee');
```
Comment (yaroslavbilokin) [inline src/redux/categories/categories.sagas.js:52]: please remove this!

File: src/routes/routes.js [hunk 8]
```diff
@@ -42,10 +44,23 @@ const Routes = () => {
     <ConnectedRouter history={history}>
       <NavBar />
       <NavMenu />
-      <Switch>
+      <Switch style={{ paddingTop: '90px' }}>
```
Comment (yaroslavbilokin) [inline src/routes/routes.js:47]: Please move this to routes.style.js

File: src/utils/client.js [hunk 9]
```diff
@@ -1,12 +1,40 @@
+import { gql } from 'apollo-boost';
```
Comment (yaroslavbilokin) [inline src/utils/client.js:1]: Please remove this! We use only apollo-client v3

**Review by bandvov — APPROVED** [review]

**Review by yaroslavbilokin — APPROVED** [review]

**Pull request merged**